src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -1157,11 +1157,20 @@ public boolean onInteract(final Player p, MultiBlock mb, final Block b) {
 							boolean craft = true;
 							for (int j = 0; j < inv.getContents().length; j++) {
 								if (!SlimefunManager.isItemSimiliar(inv.getContents()[j], inputs.get(i)[j], true)) {
-									craft = false;
-									break;
+									if (SlimefunItem.getByItem(inputs.get(i)[j]) instanceof SlimefunBackpack) {
+										if (!SlimefunManager.isItemSimiliar(inv.getContents()[j], inputs.get(i)[j], false)) {
+											craft = false;
+											break;
+										}
+									}
+									else {
+										craft = false;
+										break;
+									}
 								}
 							}
 
+
 							if (craft) {
 								final ItemStack adding = RecipeType.getRecipeOutputList(machine, inputs.get(i));
```

Comment:
You should also implement the full fix for the Enhanced Crafting Table for the Magic Workbench, as it lacks the second and destroys the id and the backpack contents.

https://github.com/mrcoffee1026/Slimefun4/blob/7d001f2b69b5d21301717ea0375122cc97277521/src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java#L232-L284